provisioning/playbook.yml

```diff
@@ -29,9 +40,43 @@
         update_cache: yes
         cache_valid_time: 300
         upgrade: yes
+    # Create fake let's encrypt directories when in development
```

Comment:
Sorry I implemented exactly the same thing on master so this shouldn't be necessary anymore. See https://github.com/openaustralia/morph/commit/6e59591dc6a196abb2e11cbec4cb9642da35bad7